### ISSUE TITLE
bdr_subscribe weirdness about pg_restore version

### ISSUE THREAD

**Issue by jyang** [2ndQuadrant/bdr#86]
First of all, thank you for this product! I was excited to find UDR which is what I need and easy to set up.

Well, I am almost there but encountered this error when doing bdr_subscribe:

bdr: failed to find pg_restore relative to binary /usr/lib/postgresql/9.4/bin/postgres or wrong version (expected 9.4.2)

The weird thing is, I have both 9.4.3 on the master and the slave. Why is 9.4.2 expected? Is this a bug?

**Comment by ringerc** [comment]
How did you install UDR? And how'd you install 9.4.3?

This seems related to #75, suggesting that this might be more of an issue with UDR than BDR.

**Comment by jyang** [comment]
Here is how I installed UDR.

Master:

* Installed regular PostgreSQL 9.4
* Installed [BDR extension](http://packages.2ndquadrant.com/bdr/apt/)
* shared_preload_libraries = 'bdr'
* wal_level = logical
* max_wal_senders = 1
* max_replication_slots = 1

Slave:

* Installed regular PostgreSQL 9.4
* Installed [BDR extension](http://packages.2ndquadrant.com/bdr/apt/)
* shared_preload_libraries = 'bdr'
* Created database with a name same as the one to be replicated from master (say, "test")
* create extension btree_gist;
* create extension bdr;
* SSH forward localhost:15203 to master:5433
* SELECT bdr.bdr_subscribe(local_node_name := '<master>', subscribe_to_dsn := 'host=localhost port=15203 user=postgres password=postgres dbname=test', node_local_dsn := 'port=5433 dbname=test');

Above call on bdr_subscribe gave:

bdr: failed to find pg_restore relative to binary /usr/lib/postgresql/9.4/bin/postgres or wrong version (expected 9.4.2)

**Comment by ringerc** [comment]
OK, so you're using BDR in UDR mode.

How was 9.4 installed - from packages, or source? Same for BDR. Show the commands.

**Comment by jyang** [comment]
9.4 was installed from package:

sudo apt-get install postgresql-9.4

**Comment by jyang** [comment]
Any further comment? Thanks!

**Comment by martinmarques** [comment]
I think Craig asked how bdr was installed as well.

If from source, paste the actual commands executed.

**Comment by jyang** [comment]
I installed UDR by installing BDR according to instructions on this page:

http://packages.2ndquadrant.com/bdr/apt/

whose commands I copy here:

```
wget --quiet -O - http://packages.2ndquadrant.com/bdr/apt/AA7A6805.asc | sudo apt-key add -
sudo apt-get update
sudo apt-get install postgresql-9.4-udr-plugin
```

**Comment by PJMODOS** [comment]
The issue is with version comparison, it's same issue as Josh described in #75. Basically the 9.4.2 UDR package will only work with Postgres 9.4.2 and no other version.

This was fixed by commit d28cfc5 but the fixed package wasn't released yet.

**Comment by ringerc** [comment]
BDR 0.9.2 contains this fix and is tagged and released.
